api/db/changelog/db.changelog-12.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+    <changeSet author="brianfreeman" id="changelog-12">
+
+        <createTable tableName="participant_cohort_status">
+            <column name="cohort_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="cdr_version_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="participant_id" type="BIGINT">
```

Comment:
I'm wondering if where I had "participant" we should be using "person" as that's the OMOP term. (I have no strong feeling either way.)

Reply:
I think we will use participant in the review UI. So I think participant is fine.